pkg/ctl/delete/nodegroup.go

```diff
@@ -62,8 +61,8 @@ func doDeleteNodeGroup(cmd *cmdutils.Cmd, ng *api.NodeGroup, updateAuthConfigMap
 		return err
 	}
 
-	if err := ctl.RefreshClusterConfig(cfg); err != nil {
-		return errors.Wrapf(err, "getting credentials for cluster %q", cfg.Metadata.Name)
+	if ok, err := ctl.CanOperate(cfg); !ok {
+		return err
```

Comment:
Don't we need the wrap message we had before?

Reply:
No, because new one is more meaningful...